Add bcdedit trick for hypervisor troubleshooting (#11483)
docker/docker.github.io: docker-for-windows/troubleshoot.md

#### Hypervisor enabled at Windows startup

If you have completed the steps described above and are still experiencing
Docker Desktop startup issues, this could be because the Hypervisor is installed,
but not launched during Windows startup. Some tools (such as older versions of 

> Virtual Box) and video game installers require you to manually launch the Hypervisor 
> at Windows startup. To do this:

Author: ```suggestion
Virtual Box) and video game installers disable hypervisor on boot. To reenable it:
```
